### DISCUSSION TITLE
runtime error: CUDA out of memory

### DISCUSSION THREAD

**Discussion by kmax2001** [raisimTech/raisimLib#42] [Q&A]
I often have this error. 
RuntimeError: CUDA out of memory. Tried to allocate 24.00 MiB (GPU 0; 1.96 GiB total capacity; 155.58 MiB already allocated; 25.00 MiB free; 174.00 MiB reserved in total by PyTorch).
May I know how can I solve this problem? Should I change the num_threads value? If so, what value will be appropriate?

**Comment by jhwangbo** [comment]
The cause seems apparent. Reducing the numner of threads will help but it depends on your environment

**Reply by kmax2001** [reply]
You told me it's because of my environment. So, is it a problem about my computer ability?

**Reply by jhwangbo** [reply]
It is about both. Your environment is using more memory than your GPU has. Either you should get a better gpu or reduce your environment
